src/fr/composants/Case-a-cocher/cas-dutilisation.md

```diff
@@ -31,7 +31,7 @@ Utilisez la case à cocher avec un [jeu de champs]({{ links.fieldset }}) pour po
 
   <h2 class="mt-0 mb-400">Composants connexes</h2>
 
-Utilisez les <a href="{{ links.radio }}" class="link-light">boutons radio</a> pour obtenir une seule réponse parmi une liste d’options.
+Utilisez les <a href="{{ links.radioGroup }}" class="link-light">Groupe de boutons radio</a> pour solliciter un choix unique parmi un plus grand ensemble d’options.
```

Comment:
Is `Groupe de boutons radio` here supposed to be capitalized? It reads as: 
> Utilisez les <a href="{{ links.radioGroup }}" class="link-light">Groupe de boutons radio</a> pour solliciter un choix unique parmi un plus grand ensemble d’options.

Reply:
No, it's not supposed to be capitalized! It should read: `Utilisez les groupes de boutons radio`
(note the 's' is currently missing for 'groupe' in this preview)